docs/details/commands/pmm-admin.md

```diff
@@ -77,6 +77,9 @@ PMM communicates with the PMM Server via a PMM agent process.
 `--group=<group-name>`
 : Group name for external services. Default: `external`
 
+`--expose-exporter-address` (This flag is availble starting with PMM 2.41.0.)
```

Comment:
```suggestion
`--expose-exporter` (This flag is availble starting with PMM 2.41.0.)
```

We figured `--expose-exporter-address` felt too long, so we went with `--expose-exporter`